### PR TITLE
fix: Get rid of two labels

### LINKED ISSUE #31289: [Bug]: Get rid of two labels
https://theappsmith.slack.com/archives/C04P60V0VQE/p1707308259360009

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #31289 

## Summary by CodeRabbit

- **New Features**
	- Ensured certain experimental features ("ab_wds_enabled" and "release_anvil_enabled") are always enabled for testing purposes.

- **Refactor**
	- Improved the reliability of widget configuration updates by ensuring modifications are made on a cloned object, preserving the original data integrity.

### CODE REVIEW

**Comment by jsartisan** [comment]
@riodeuno @KelvinOm  I found the steps to repro the issue. You will need to drop keyvalue widget and input widget on the canvas. The second widget will show the problems of two labels. The problem was that `mergeWidgetConfig` was updating the source object/array due to which second object was getting modified config.

File: app/client/src/utils/helpers.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ import { getContainerIdForCanvas } from "sagas/WidgetOperationUtils";
 import scrollIntoView from "scroll-into-view-if-needed";
 import validateColor from "validate-color";
 import { CANVAS_VIEWPORT } from "constants/componentClassNameConstants";
+import { klona as clone } from "klona/full";
```
Comment (KelvinOm) [inline app/client/src/utils/helpers.tsx:41]: Just wondering why not just use [lodash](https://lodash.com/docs/4.17.15#cloneDeep)?

Comment (jsartisan) [reply app/client/src/utils/helpers.tsx:41]: Klona clone is faster than lodash. In the past there were issues in performance due to lodash`s clone.

**Review by KelvinOm — APPROVED** [review]

File: app/client/src/ce/selectors/featureFlagsSelectors.ts [hunk 2]
```diff
@@ -12,6 +12,9 @@ export const selectFeatureFlagCheck = (
 ): boolean => {
   const flagValues = selectFeatureFlags(state);
 
+  if (flagName === "ab_wds_enabled") return true;
+  if (flagName === "release_anvil_enabled") return true;
```
[the remark this answers is not in the document]
Comment (riodeuno) [reply app/client/src/ce/selectors/featureFlagsSelectors.ts:16]: @jsartisan Shall we remove these flag hardcoding?

**Comment by riodeuno** [comment]
@jsartisan The changes look fine to me. We might want to remove the hardcoded flag values.

**Comment by jsartisan** [comment]
@riodeuno Yes, those are only for testing on the DP. I'll remove it now.

**Review by riodeuno — APPROVED** [review]

**Review by KelvinOm — APPROVED** [review]

**Pull request merged**